impl/implstream/kafkaconfluent/stream.go

```diff
@@ -7,11 +7,22 @@ import (
 	"sync"
 	"time"
 
-	"github.com/arquivei/foundationkit/errors"
 	"github.com/arquivei/goduck"
-	"github.com/rs/zerolog/log"
 
+	"github.com/arquivei/foundationkit/errors"
 	"github.com/confluentinc/confluent-kafka-go/kafka"
+	"github.com/rs/zerolog/log"
+)
+
+var (
+	//ErrEmptyBroker is returned when the the broker is missing from the Config struct
+	ErrEmptyBroker = errors.New("invalid config: empty broker")
```

Comment:
Nit: I think we used to favour `bad` instead of `invalid` in error messages

Reply:
done